numbers/guides/downloadingReports.md

```diff
@@ -0,0 +1,106 @@
+| INVOICE              | A copy of the invoice file or files for the specified date range                                                                                     |
+| STMTBDR              | BDR records that are aligned with the invoice for Voice Services                                                                                     |
+| DIDSNAP              | A list of telephone numbers, aligned with invoice                                                                                                    |
+| DIDSNAP_MSG          | A list of telephone numbers that are enabled for messaging, aligned with invoice                                                                     |
+| RECORDINGBDR         | Billing Detail Records for Recording Services - per recording information, available on day to day basis                                             |
+| RECORDINGSTMTBDR     | BDR records that are aligned with the invoice for Recording Services                                                                                 |
+| TRANSCRIPTIONBDR     | Billing Detail Records for Transcription Services - per call transcription information, available on day to day basis                                |
+| TRANSCRIPTIONSTMTBDR | BDR records that are aligned with the invoice for Recording Services                                                                                 |
+| CNAMBDR              | Record of CNAM Subscribe and Notify queries/dips, available on day to day basis                                                                      |
+| CNAMSTMTBDR          | Record of CNAM Subscribe and Notify queries/dips, aligns with invoices                                                                               |
+| CONFBDR              | Billing Detail Records for Transcription Services - per conference information, available on day to day basis                                        |
+| CONFSTMTBDR          | BDR records that are aligned with the invoice for Conferencing Services                                                                              |
+
+{% extendmethod %}
+## Request a Report {#request-report}
+Creating a report starts with a <code class="post">POST</code> request to `https://dashboard.bandwidth.com/api/accounts/{accountId}/billingReports`, detailing the date range for the report as well as type of report you wish to download. A successful response will include a body and location header, which contains a URI with the report ID to be used in later requests.
+
+{% common %}
+### Request
+```http
+POST https://dashboard.bandwidth.com/api/accounts/{accountId}/billingReports HTTP/1.1
+Content-Type: application/xml; charset=utf-8
+Authorization: Basic dXNlcm5hbWU6cGFzc3dvcmQ=
+
+<BillingReport>
+    <Type>BDR</Type>
+    <DateRange>
+        <StartDate>2020-05-21</StartDate>
```

Comment:
Is this the date time format correct?  We really should have one date-time format for our whole api

Reply:
That is the correct date and time format, which I don't believe matches up with some of the other endpoints